ios/swift/FriendlyChatSwift.xcodeproj/project.pbxproj

```diff
@@ -187,11 +191,28 @@
 				10AF4CB51DD236DE0096D192 /* Images.xcassets in Resources */,
 				107960731C612531003DB6CC /* Main.storyboard in Resources */,
 				10AF4CB31DD236DE0096D192 /* logo1024.png in Resources */,
+				779FCD711FDF0B8B00582829 /* GoogleService-Info.plist in Resources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		779FCD6F1FDF0B2F00582829 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${PODS_ROOT}/Fabric/run\"";
+		};
+/* End PBXShellScriptBuildPhase section */
+
```

Comment:
I believe this should be deleted as well. could be added as intruction if this script has to be added manually.

Reply:
Yeah realized that script here is fine.